great_expectations/render/renderer/site_builder.py

```diff
@@ -410,6 +424,9 @@ def build(self):
         resource_keys = target_store.list_keys()
         index_links_dict = OrderedDict()
 
+        if self.show_cta_footer:
```

Comment:
`show_cta_footer` comes from new key in config file, which gets passed to index builder when site builder instantiates it